pkg/controllers/dynakube/oneagent/oneagent_reconciler.go

```diff
@@ -107,6 +115,15 @@ func (r *Reconciler) Reconcile(ctx context.Context) error {
 		return r.cleanUp(ctx)
 	}
 
+	log.Info("creating Pull Secret")
```

Comment:
do we need this log line here?
(you don't have it in the AG reconciler for the same logic)